haskell/src/Lorentz/Contracts/Stablecoin.hs

```diff
@@ -2,7 +2,91 @@
 -- SPDX-License-Identifier: MIT
 
 module Lorentz.Contracts.Stablecoin
-    (
-    ) where
+  ( Parameter
+  , Storage
+  , stablecoinPermissionsDescriptor
+  , stablecoinTokenMetadata
+  ) where
+
+import Lorentz
+import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
+import Lorentz.Contracts.Spec.FA2Interface (OperatorTransferMode(..), OwnerTransferMode(..), SelfTransferMode(..))
+import qualified Lorentz as L
+import Util.Named
+
+-- Parameter
+type Parameter = FA2.Parameter
+
+-- Permissions descriptor
+data OwHookOptReq = OptOH () | ReqOp ()
```

Comment:
Are these `()`s required? I thought they are implied automatically if they are missing :thinking:

Reply:
You are right. Removed them.